src/index.ts

```diff
@@ -227,12 +259,22 @@ export class Amplitude {
    * @param groupType
    * @param groupName
    * @param identifyInstance
+   * @param extra
    */
   groupIdentify(
     groupType: string,
     groupName: string | string[],
     identifyInstance: Identify,
+    extra?: MiddlewareExtra,
   ): Promise<boolean> {
+    const event = {
+      event_type: SpecialEventType.GROUP_IDENTIFY,
+      group_properties: identifyInstance,
```

Comment:
I guess same concern for group properties

Reply:
fixed (deep clone)